packages/kotti-ui/source/kotti-accordion/KtAccordion.vue

```diff
@@ -1,145 +1,138 @@
 <template>
-	<div class="accordion">
-		<div
-			class="accordion__header"
-			:class="{ 'accordion__header--clickable': isFullyClickable }"
-			@click="headerToggle"
-		>
-			<div class="accordion__title">
+	<div class="kt-accordion">
+		<div :class="headerClasses" @click="headerToggle">
+			<div class="kt-accordion__title">
 				<slot name="title">
-					<i v-if="icon" class="yoco accordion__title__icon" v-text="icon" />
-					<div class="accordion__title__text" v-text="title" />
+					<i v-if="icon" class="yoco kt-accordion__title__icon" v-text="icon" />
+					<div class="kt-accordion__title__text" v-text="title" />
 				</slot>
 			</div>
-			<div class="accordion__toggle" @click.stop="toggle">
-				<i class="yoco" v-text="isOpen ? 'minus' : 'plus'" />
+			<div class="kt-accordion__toggle" @click.stop="toggle">
+				<i class="yoco" v-text="toggleIcon" />
 			</div>
 		</div>
-		<div
-			class="accordion__content"
-			:class="isOpen ? 'is-open' : 'is-close'"
-			:style="`--height: ${contentHeight}px`"
-			@click="setHeight"
-		>
-			<div ref="contentInner" class="inner">
+		<div :class="contentClasses">
+			<div ref="contentInner" class="kt-accordion__content__inner">
 				<slot />
 			</div>
 		</div>
 	</div>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { Yoco } from '@3yourmind/yoco'
+import { computed, defineComponent, ref } from '@vue/composition-api'
+
+import { useSlideAnimaion } from './hooks'
+import { KottiAccordion } from './types'
+
+export default defineComponent<KottiAccordion.PropsInternal>({
 	name: 'KtAccordion',
 	props: {
 		icon: { default: null, type: String },
 		isClosed: { default: false, type: Boolean },
 		isFullyClickable: { default: false, type: Boolean },
```

Comment:
I think we should actually drop support and make this the default. From what I can tell, there's no good reason why we wouldn't want to always have this set to true. Right now, AFAIK, it's unused as well…

Reply:
double this opinion